### PR TITLE
update instructions

### PR DESCRIPTION
**Pull request by MattDHill**

### CODE REVIEW

File: instructions.md [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
+
+**Note**: It is recommended that you open a new browser tab to your Embassy while following these instructions. Otherwise, you will have to close and re-open them multiple times.
+
+## Initial Setup
+
+1. In Config, choose whether you want to use CryptPad over LAN (.local) or Tor (.onion). This can be changed at any time.
+
+**Note**: *embassyOS is unaware of the above setting. For example, if you are currently connected to your Embassy over Tor and you click "Launch UI", CryptPad will open CryptPad over Tor, even if you have CryptPad configured to use LAN. You will have to manually edit the URL bar in this case.*
+
+2. Launch your CryptPad UI, click "Sign Up", and create an account. Choose a good password. We recommend saving this password in a password manager, such as Vaultwarden on you Embassy.
```
Comment (chrisguida) [inline instructions.md:11]:
```suggestion
2. Launch your CryptPad UI, click "Sign Up", and create an account. Choose a good password. We recommend saving this password in a password manager, such as Vaultwarden on your Embassy.
```

File: instructions.md [hunk 2]
```diff
@@ -1,22 +1,23 @@
...
+
+3. After creating your account, click your name abbreviation in the upper right corner. Click Profile --> View My Profile. Copy the public key from your profile page, then go back into CryptPad config on your Embassy and paste this public key into "Admin User Public Key". This grants your user the ability to access the Admin Panel, where you can manage more configurations - including preventing any new users from signing up.
+
+4. Optionally enter an email address into your CryptPad config settings. If you choose to share your CryptPad servers with others, this is the email they will see to contact you for assistance.
+
+## General Usage
+
+Check out the [CryptPad docs](https://docs.cryptpad.org) for detailed instructions on how to use CryptPad
```
Comment (chrisguida) [inline instructions.md:19]:
```suggestion
Check out the [CryptPad docs](https://docs.cryptpad.org) for detailed instructions on how to use CryptPad.
```

for consistency :)

**Review by chrisguida — CHANGES_REQUESTED** [review]
Couple nits

**Review by chrisguida — APPROVED** [review]
Looks good, thanks!

**Review by islandbitcoin — COMMENTED** [review]
lgtm

**Pull request merged**